composer.json

```diff
@@ -17,7 +17,8 @@
 		"cakephp/cakephp": "^4.0.0"
 	},
 	"require-dev": {
-		"fig-r/psr2r-sniffer": "dev-master"
+		"fig-r/psr2r-sniffer": "dev-master",
+		"phpunit/phpunit": "~8.5.0"
```

Comment:
we dont need this if we use the documented composer commands.

Reply:
not required for travis, but it's required to develop the code in this repo, no? is it usual practice to keep this out?

Reply:
See composer scripts section.